Simplified.xcodeproj/project.pbxproj

```diff
@@ -3755,7 +3793,8 @@
 				);
 				PRODUCT_NAME = SimplyE;
 				SDKROOT = iphoneos;
-				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_COMPILATION_MODE = wholemodule;
+				SWIFT_OPTIMIZATION_LEVEL = "-O";
```

Comment:
I guess this was Xcode doing its... magic?

Reply:
I believe so...